packages/providers/upload-local/lib/index.js

```diff
@@ -5,6 +5,7 @@
  */
 
 // Public node modules.
+const { pipeline } = require('stream');
```

Comment:
is this available on all our node supported versions ?

Reply:
Yes :)